aggregator/src/aggregator/error.rs

```diff
@@ -103,11 +103,8 @@ pub enum Error {
     #[error("HTTP client error: {0}")]
     HttpClient(#[from] reqwest::Error),
     /// HTTP server returned an error status code.
-    #[error("HTTP response status {problem_details}")]
-    Http {
-        problem_details: Box<HttpApiProblem>,
-        dap_problem_type: Option<DapProblemType>,
-    },
+    #[error("HTTP response status {0}")]
+    Http(Box<HttpErrorResponse>),
```

Comment:
Q: why does this need to be `Box`ed?

Reply:
Note that we previously boxed `HttpApiProblem`. It's a big struct, since it holds four `String`s, a `HashMap`, and a `u16`, for a total of 152 bytes including padding. Our various error enums got flagged with `clippy::large_enum_variant` at one time before we boxed this. Keeping the `Error` enum smaller means it can be copied around with a few inline moves as it's returned in results, rather than needing to call out to memcpy.